go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -1118,7 +1118,7 @@ func (tsv *TabletServer) ExecuteBatch(ctx context.Context, target *querypb.Targe
 	}
 
 	if asTransaction {
-		transactionID, err = tsv.Begin(ctx, target, options)
+		transactionID, _, err = tsv.Begin(ctx, target, options)
```

Comment:
return alias?

Reply:
this transactionID is never returned - the tx is either committed or rolled back in the scope of the method. given that, I don't think we need to return the alias, right?

Reply:
Agreed.